codespell_lib/data/dictionary.txt

```diff
@@ -2914,7 +2914,7 @@ backwords->backwards, backwords,
 backwrad->backward
 bactracking->backtracking
 bacup->backup
-bae->base
+bae->base, disabled due to being a person's surname
```

Comment:
Personally surnames seem like a slippery slope to me. I can just about see first names, but the range of surnames is far greater. TBH both first and surnames feel like local whitelists per project really. Or again a separate dictionary for them.

Reply:
That's a good point.

My argument for including these names is:
 - The number of common misspellings that are also common names is actually not too large, and is much smaller than the list of all names.
 - It can be pretty annoying if your name gets *automatically* "corrected" after being flagged as a misspelling. Furthermore, if it was a traditional English name like Johnson or Smith it would never be auto-corrected, but if you have the surname Chang, Wen, Bae etc. (each not too rare in different parts of Asia) then you'd get flagged, which seems slightly unfair.

Reply:
Fair point, I'd suggest in this particular case, bae could perhaps be disabled due to being a slang term of endearment, and the name of a large company (BAE Systems; although as below I'd prefer to avoid that), primarily before you hit the surname aspect.

If you take "variants" of my name, then there are Peter, Pete and Petr that I'm immediately aware of; Petr could be a typo of Petri (dish).

Off the top of my head, and perhaps not a great example, Smyth ( https://en.wikipedia.org/wiki/Smyth ) is a surname, but also possibly a typo of Smith (the keys are even in similar places to be a plausible typo).

I take your point though, but to me this feels like another strong use-case for the multi-dictionary approach I mentioned in https://github.com/codespell-project/codespell/issues/103#issuecomment-354913920 .

Reply:
Done in #1476.